HLTrigger/Egamma/plugins/HLTScoutingEgammaProducer.cc

```diff
@@ -146,16 +187,55 @@ void HLTScoutingEgammaProducer::produce(edm::StreamID sid, edm::Event& iEvent, e
     if (candidateRef.isNull() && !candidateRef.isAvailable())
       continue;
 
-    if ((*HoverEMap)[candidateRef] > egammaHoverECut)
-      continue;
     if (candidate.pt() < egammaPtCut)
       continue;
     if (fabs(candidate.eta()) > egammaEtaCut)
       continue;
 
+    edm::ESHandle<CaloTopology> pTopology;
+    setup.get<CaloTopologyRecord>().get(pTopology);
+    const CaloTopology* topology = pTopology.product();
```

Comment:
Please declare the consumption of EventSetup products
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#In_ED_module
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#Getting_data_from_EventSetup_wit

Reply:
would there already be an example for CaloTopology available?

Reply:
`git grep "ESGetToken<CaloTopology,"` shows many, e.g. https://github.com/cms-sw/cmssw/blob/master/Calibration/HcalCalibAlgos/plugins/HcalIsoTrkAnalyzer.cc

But it's really just something along
```cpp
// in class declaration
edm::ESGetToken<CaloTopology, CaloTopologyRecord> topologyToken_;

// in constructor
topologyToken_ = esConsumes();

// here
const CaloTopology* topology = &setup.getData(topologyToken_);
```